emscripten/luna-utils.js

```diff
@@ -0,0 +1,46 @@
+var Module = { 'preRun': function() {
+
+callLuna = function(luaFileName, tnsFileName, cb)
+{
+    var ret = Module.callMain([luaFileName, tnsFileName]);
```

Comment:
This code looks like it got infected by the node.js callback virus. IMO that does not make any sense here, what about the simple linear approach directly in fileLoaded?